bitbots_body_behavior/bitbots_body_behavior/body_behavior.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, node:Node):
         self.dsd.blackboard.blackboard.head_pub = node.create_publisher(HeadMode, "head_mode", 10)
         self.dsd.blackboard.pathfinding.direct_cmd_vel_pub = node.create_publisher(Twist, 'cmd_vel', 1)
         self.dsd.blackboard.pathfinding.pathfinding_pub = node.create_publisher(PoseStamped, 'goal_pose', 1)
-        self.dsd.blackboard.pathfinding.pathfinding_cancel_pub = node.create_publisher(Empty, 'move_base/cancel', 1)
+        self.dsd.blackboard.pathfinding.pathfinding_cancel_pub = node.create_publisher(Empty, 'move/cancel', 1)
```

Comment:
Sadly I think this is still the topic name for it in ROS2. Also we would need to change our path planing if we change the topic name.

Reply:
could I just change the topic name in hcm_blackboard.py in line 123?